doc/cmd_run.rst

```diff
@@ -271,6 +271,42 @@ from a task while it is being executed,
 this is controlled by the ``verbosity`` param.
 
 
+check_file_uptodate
+-------------------
+
+`doit` provides different options to check if files are up to date.
+Use the option ``--check_file_uptodate`` to choose:
+
+ * default: use the timestamp, size and md5sum of the file.
```

Comment:
I think we could say that the default is md5sum, the usage of timestamp and size is just an implementation detail for speed optimization. So just say it is md5 and then put a note about the optimization.